Calibration/HcalCalibAlgos/plugins/HcalHBHEMuonAnalyzer.cc

```diff
@@ -183,43 +191,49 @@ HcalHBHEMuonAnalyzer::~HcalHBHEMuonAnalyzer() {
 void HcalHBHEMuonAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup) {
   ++kount_;
   clearVectors();
-  RunNumber   = iEvent.id().run();
-  EventNumber = iEvent.id().event();
-  LumiNumber  = iEvent.id().luminosityBlock();
-  BXNumber    = iEvent.bunchCrossing();
-#ifdef DebugLog
-  edm::LogInfo("HcalHBHEMuon") << "Run " << RunNumber << " Event " << EventNumber
-			       << " Lumi " << LumiNumber << " BX " << BXNumber;
+  int depthHE = (maxDepth_ <= 6) ? 3 : 4;
```

Comment:
@bsunanda could you add a comment on these magic numbers, please
Or better use constants with a meaningful name.